### PR TITLE
Dispatch app event `settings` when Settings Charm activated

### PR DESCRIPTION
**Pull request by donavon**

### CODE REVIEW

File: lib/app.js [hunk 1]
```diff
@@ -15,12 +15,12 @@ exports.start = function () {
     WinJS.Binding.optimizeBindingReferences = true;
 
     app.addEventListener("activated", function (args) {
+        //see http://msdn.microsoft.com/en-us/library/windows/apps/windows.applicationmodel.activation.activationkind
```
Comment (domenic) [inline lib/app.js]: Not so sure about this section---at the very least belongs in a separate commit. But I got the impression there were other kinds of activation we should probably handle, or at least not break WinningJS apps from handling themselves.

**Comment by domenic** [comment]
Look at all the suck you've removed from WinJS!! _This_ is what WinningJS is about.

I was about to comment that an object with an `append` property would be nicer, but then you did that as I was typing, so :D

File: lib/app.js (hunk 1 above)
Comment (donavon) [inline lib/app.js]: accidentally made it into this pull request. i'll remove and make it separately.

**Comment by pbouzakis** [comment]
Looks good! Merge it!
